### ISSUE TITLE
[v3] Don't assume icons are square

### ISSUE THREAD

**Issue by aidanlister** [coreui/coreui#105]
```
.btn i, .btn .c-icon {
    width: 0.875rem;
}
```

<img width="327" alt="Screen Shot 2019-12-20 at 10 08 34 am" src="https://user-images.githubusercontent.com/54896/71224468-bf0d7300-2310-11ea-9289-14ca7c3076e3.png">

**Comment by mrholek** [comment]
Fixed in v3.0.0-rc.3

**Comment by aidanlister** [comment]
I don't think this is fixed ... I'm still seeing it in the compiled source. I couldn't see it in _buttons.scss but I found it in the mixins:

```
// Button sizes
@mixin button-size($padding-y, $padding-x, $font-size, $line-height, $border-radius) {
  padding: $padding-y $padding-x;
  @include font-size($font-size);
  line-height: $line-height;
  // Manually declare to provide an override to the browser default
  @include border-radius($border-radius, 0);

  i,
  .c-icon {
    width: $font-size;
    height: $font-size;
    margin: ((($font-size * $line-height) - $font-size) / 2) 0;
  }
}
```

https://github.com/coreui/coreui/blob/master/scss/mixins/_buttons.scss#L141
